doc/api/fs.md

```diff
@@ -1640,7 +1640,9 @@ Read data from the file specified by `fd`.
 `length` is an integer specifying the number of bytes to read.
 
 `position` is an integer specifying where to begin reading from in the file.
-If `position` is `null`, data will be read from the current file position.
+If `position` is `null`, data will be read from the current file position, 
+and the file position will be updated for subsequent reads.
+If `position` is an integer, the file position will remain unchanged.
```

Comment:
maybe it feels obvious, but for null you say where it reads from (current position) and what happens to current position (its updated). For the integer case this says file position is unchange, but it doesn't say where read is from, which is from the integer as a positive offset from beginning of file (-4 won't  read from 4 bytes before end of file, for example).

also, the file position will be changed on Windows, though there is a UV PR to change windows to be like unix: https://github.com/libuv/libuv/pull/1357, not sure if that has effected node yet, though.